lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_germany_when_residing_in_the_uk.govspeak.erb

```diff
@@ -30,11 +30,7 @@
     <%= render partial: 'partner_naturalisation_in_uk.govspeak.erb' %>
   <% end %>
 
-  <%= render partial: 'consular_fees_table.govspeak.erb',
-             locals: { calculator: calculator } %>
-  <%= render partial: 'link_to_consular_fees.govspeak.erb',
-             locals: { calculator: calculator } %>
-  <%= render partial: 'payment_information.govspeak.erb',
+  <%= render partial: 'os_affirmation/fees.govspeak.erb',
```

Comment:
I've updated the commit note to make it clearer that it's changing two outcomes.